[DOCS-1263] Update docs for Org Dashboard (#1201)
wandb/docs: content/guides/hosting/monitoring-usage/org_dashboard.md

- **Teams**: lists details about each team, including the number of users and tracked hours, and allows an admin to join a team.
- **Billing**: ssummarizes your organization's charges, allows you to run and export billing reports, and shows details about your license, such as when it expires.
- **Settings**: allows you to configure custom roles and settings related to privacy and authentication.

## View the status of a user
The **Users** tab lists all users, along with data about each user. The **Last Active** column shows whether a user has accepted an invitation and the user's current status:

* **Invite pending**: Admin has sent invite but user has not accepted invitation. 
* **Active**: User has accepted the invite and created an account.
* **-**: The user was previously active but has not been active in the last 6 months.
* **Deactivated**: Admin has revoked access of the user.

To sort the list of users by activity, click the **Last Active** column heading.

## View and share how your organization uses W&B
From the **Users** tab, view details about how your organization uses W&B in CSV format.

1. Click the action `...` menu next to the **Invite new user user** button.

> 2. Click **Export as CSV**. A CSV file downloads. It lists details about each user of an organization, such as their user name and email address, the time they were last active, their roles, and more.

Reviewer: ```suggestion
2. Click **Export as CSV**. The CSV files that downloads lists details about each user of an organization, such as their user name and email address, the time they were last active, their roles, and more.
```
